tests/mpi/tests/rm_execution_single_host

```diff
@@ -12,7 +12,7 @@ check_rms
 RMS=$RESOURCE_MANAGER
 
 NODES=1
-TASKS=2
+TASKS=`tasks_count 2`
```

Comment:
Which .spec should be added to `spec_to_test_mapping.py` for this test ?

Reply:
probably openmpi4.spec, mpich.spec and mvapich2.spec

Reply:
I've added mappings for `openmpi4` and `mpich`.
`mvapich2` requires InfiniBand and thus cannot be tested on Github Actions.